src/nimble.nim

```diff
@@ -1190,7 +1222,9 @@ proc doAction(options: var Options) =
     var pkgInfo = getPkgInfo(getCurrentDir(), options)
     publish(pkgInfo, options)
   of actionDump:
-    dump(options)
+    dump(options, json = false)
+  of actionDumpJson:
+    dump(options, json = true)
```

Comment:
Instead of `dumpJson` can we get a `--json` flag for ``dump``?

Reply:
done